### PR TITLE
[#29] Update readme to 0.6.0

### LINKED ISSUE #29: Updating README to version 0.6

### PR DESCRIPTION
**Pull request by spi-x-i**
You can see preview of this [here](https://github.com/spi-x-i/flink-jpmml/tree/feature/%2329-update-README-to-0.6.0).
This closes #29

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,11 +65,88 @@ Lets start.
...
+
+Since `0.6.0` the project supports dynamic streaming model serving efficiently. For more information 
+we suggest to watch the [related talk](https://www.youtube.com/watch?v=0rWvMZ6JSD8&t=7s) 
+presented at last Flink Forward 2017 in Berlin.
+
+First of all, we indentify univocally models by the `ModelId` abstraction, made of an *applicationName*
+and a *version*. 
+
+> **e.g.** Suppose you have two _ensemble_ models A and B (PMML based) where A has a depth level 10 on width 
+100 and B depth 5 on width 200, and you desire to have a comparison between them, so likely your can identify
```
Comment (riccardo14) [inline README.md]: your -> you

File: README.md [hunk 2]
```diff
@@ -65,11 +65,88 @@ Lets start.
...
+
+- `DataStream[ServingMessage]` this stream is the main user tool to feed the operator with
+necessary model information; here, the user is not demanded to send by stream its PMML models but only the requested 
+descriptive metadata. The user should employ `ServingMessage` ADT in order to feed this stream. 
+By now, the user can define the following two messages:
+    - `AddMessage` it requires an `applicationName` Java UUID formatted, a `version`, the model source
+    `path` and a timestamp
+    - `DelMessage` it requires an `applicationName` Java UUID formatted, a `version` and a timestamp
+- `DataStream[BaseEvent]` your input stream should extend the `BaseEvent` trait and defining the string `modelId
+ formatted as `"<modelApplication>_<modelVersion>"` and a timestamp.
```
Comment (riccardo14) [inline README.md]: typo `

**Review by riccardo14 — CHANGES_REQUESTED** [review]

**Review by riccardo14 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -13,13 +13,13 @@ scenario.
...
 * any well-known version of a PMML model (**3.2** or above)
-* flink-jpmml is tested with the latest Flink (i.e. 1.3.1), but any working Apache Flink version 
+* flink-jpmml is tested with the latest Flink (i.e. 1.3.2), but any working Apache Flink version 
 ([repo](https://github.com/apache/flink)) should work properly.
 
 ## Adding `flink-jpmml` dependency
 * if you employ sbt add the following dependecy to your project:
-    * Snapshot: `"io.radicalbit" %% "flink-jpmml-scala" % "0.6.0-SNAPSHOT"`
-    * Stable: `"io.radicalbit" %% "flink-jpmml-scala" % "0.5.1"`
+    * Snapshot: `"io.radicalbit" %% "flink-jpmml-scala" % "0.7.0-SNAPSHOT"`
```
Comment (maocorte) [inline README.md:21]: Should be also updated the project version into `version.sbt` reflecting the changes

**Review by maocorte — COMMENTED** [review]
maybe `architecture.png` could be moved into a dedicated folder (for example `images/`) so it is not included in the builded jar

**Review by maocorte — APPROVED** [review]

**Review by maocorte — APPROVED** [review]

**Pull request merged**